components/raftstore-v2/src/raft/storage.rs

```diff
@@ -184,6 +184,16 @@ impl<ER: RaftEngine> Storage<ER> {
     pub fn is_initialized(&self) -> bool {
         self.region_state.get_tablet_index() != 0
     }
+
+    #[inline]
+    pub fn applied_term(&self) -> u64 {
```

Comment:
I don't think it's necessary. If `applied_term` needs to be wrapped, then every access method in `EntryStorage` also needs to be wrapped. It makes no sense.

Reply:
then why raft_state and apply_state are wrapped above ?

Reply:
the reason is applied_term  will be very frequently used, just like in v1.